File: MODUL 1/PRAK110-2610817110018-Gusti_Ahdan_Ziyad.py
import math

base = 5
height = 12
side = int(math.sqrt((base * base) + (height * height)))
perimeter = base + height + side
area = int((base * height) / 2)

print("Diketahui :")
print(f"Alas = {base} cm")
print(f"Tinggi = {height} cm\n")
print("Jawab :")
print(f"Sisi A = {base} cm")
print(f"Sisi B = {side} cm")
print(f"Sisi C = {height} cm")
print(f"Keliling = {perimeter} cm")
print(f"Luas = {area} cm")

File: MODUL 1/PRAK110-2610817110018-Gusti_Ahdan_Ziyad.c
#include <stdio.h>
#define _USE_MATH_DEFINES
#include <math.h>

int main()

{
    int base = 5;
    int height = 12;
    double side = sqrt((base * base) + (height * height));
    int perimeter = base + height + side;
    int area = (base * height) / 2;

    printf("Diketahui :\n");
    printf("Alas = %d cm\n", base);
    printf("Tinggi = %d cm\n", height);
    printf("\n");
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", base);
    printf("Sisi B = %.0f cm\n", side);
    printf("Sisi C = %d cm\n", height);
    printf("Keliling = %d cm\n", perimeter);
    printf("Luas = %d cm\n", area);
    return 0;
}